tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -3712,7 +3712,7 @@ TEST_F(CompilerTest, CompileWhenODumpThenOptimizerMatch) {
     LPCWSTR Target = L"ps_6_0";
     CreateBlobFromText("float4 main() : SV_Target { return 0; }", &pSource);
 
-    LPCWSTR Args[2] = {OptLevel, L"/Odump"};
+    LPCWSTR Args[3] = {OptLevel, L"-select-validator external", L"/Odump"};
```

Comment:
`/Odump` should not run validation.  I don't understand why this option would be necessary.

Reply:
I can confirm that this change does fix this test failure. Previously there was a failure when running `RunOptimizer`.